docs/source/building/windows.rst

```diff
@@ -15,31 +15,24 @@ Install dependencies:
    .. code-block:: bash
 
       pacman -S \
-        base-devel \
-        cmake \
-        diffutils \
-        gcc \
-        git \
-        make \
-        mingw-w64-x86_64-binutils \
```

Comment:
I'm pretty sure most of these cannot be removed

Reply:
I took this replacement install command out of my bash history while setting up the ucrt64 environment. The "toolchain" package covers all of these except make. If I have to modify this command to make the actions pipeline work, I'll do so.

Git is omitted because presumably the prospective contributor already used that tool to check out the repository.

Reply:
I believe you may have missed an important dependency (at least, it didn't get installed as a co-depend of any other package on my up-to-date MSYS2 install):
```
$ pacman -Ql | grep winrt/Windows.Graphics.Capture.h
mingw-w64-ucrt-x86_64-cppwinrt /ucrt64/include/winrt/Windows.Graphics.Capture.h
```

I would also encourage you not to omit the backslashes prior to the last entry, otherwise the command cannot be copy & pasted into a terminal correctly. Also make sure not to add any comments to these lines (OT: the Linux build document breaks copy & paste for this reason).

Reply:
Also agree with @psyke83 on the back slashes, they are line continuation statements.

I also like to call out specifically what we use, and not assume they will be included as a sub-dependency. Even if they are included today, there's no guarantee they'll be included in future versions.

For git, it is required for our versioning inside of cmake. Even if they git installed, there's no promise that msys2 shell will pick up the version that was used.

Reply:
Noted. I'll be adding git and cppwinrt to the list of dependencies. The others are either explicitly called out when installing `mingw-w64-ucrt-x86_64-toolchain`, or no longer relevant (specifically `make`).

Reply:
If you do refresh your PR for whatever reason, please also remove the line break continuation from the final item in the list (currently `mingw-w64-ucrt-x86_64-toolchain`) . Leaving that causes the terminal to wait for additional input by the user. Anyway, this is such a minor issue - so let me say, nice work!